ledger/testchain-generator/Cargo.toml

```diff
@@ -22,6 +22,7 @@ workspace = true
 
 [dependencies.snarkvm-console]
 workspace = true
+features = [ "test_consensus_heights" ]
```

Comment:
Pretty sure we want this by default, otherwise we would never be able to reach the current consensus version.